content/6_Advanced/Treaps.mdx

```diff
@@ -147,50 +178,64 @@ ll ans, inv;
 
 typedef struct tnode* pt;
 struct tnode {
-	int pri, val; pt c[2]; // essential
-	int sz; // for range queries
-	tnode (int _val) {
-		pri = rng(); val = _val;
-		sz = 1; c[0] = c[1] = NULL;
+	int pri, val;
+	pt c[2];  // essential
+	int sz;  // for range queries
+	tnode (int val) {
+		pri = rng();
+		val = val;
+		sz = 1;
+		c[0] = c[1] = NULL;
 	}
 };
-int getsz(pt x) { return x?x->sz:0; }
+
+int getsz(pt x) { return x ? x->sz : 0; }
+	
 pt calc(pt x) {
-	x->sz = 1+getsz(x->c[0])+getsz(x->c[1]);
+	x->sz = 1 + getsz(x->c[0]) + getsz(x->c[1]);
 	return x;
 }
-pair<pt,pt> split(pt t, int v) { // >= v goes to the right
-	if (!t) return {t,t};
+	
+pair<pt, pt> split(pt t, int v) { // >= v goes to the right
+	if (!t) return {t, t};
 	if (t->val >= v) {
-		auto p = split(t->c[0], v); t->c[0] = p.s;
-		return {p.f,calc(t)};
+		auto p = split(t->c[0], v);
+		t->c[0] = p.s;
+		return {p.f, calc(t)};
 	} else {
-		auto p = split(t->c[1], v); t->c[1] = p.f;
-		return {calc(t),p.s};
+		auto p = split(t->c[1], v);
+		t->c[1] = p.f;
+		return {calc(t), p.s};
 	}
 }
 
 pt merge(pt a, pt b) {
-	if (!a || !b) return a?:b;
+	if (!a || !b)
+		return a ? NULL : b;
```

Comment:
no revert this change. braces or one line

Reply:
This isn't my code, you can go in and make any changes necessary